modules/deployment/versions.tf

```diff
@@ -2,6 +2,9 @@ terraform {
   required_version = ">= 0.12.0"
 
   required_providers {
-    aws = ">= 4.51"
+    aws = {
```

Comment:
IMO we should enforce TF >= 1.0 in a separate PR and use the new notation

Reply:
The pipelines fail though if I dont add the new notation due to tflint detecting this as an issue